package.json

```diff
@@ -1,21 +1,26 @@
 {
   "name": "federalist-uswds-jekyll",
   "version": "1.3.0",
-  "description": "Federalist using USWDS+Jekyll",
+  "description": "Federalist template using USWDS+Jekyll",
   "scripts": {
-    "build": "npm run build:uswds; npm run build:jekyll",
-    "build:jekyll": "bundle exec jekyll build",
-    "build:uswds": "ruby build.rb uswds;",
-    "clean": "ruby build.rb clean",
-    "federalist": "npm run build:uswds;",
-    "postinstall": "bundle install",
-    "start": "npm run build; bundle exec jekyll serve;",
+    "build": "bundle exec jekyll build",
+    "clean": "bundle exec jekyll clean",
+    "start": "bundle exec jekyll serve",
     "test": "bundle exec htmlproofer _site; npx a11y '_site/**/*.html'"
   },
-  "author": "",
+  "author": "Federalist",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/18F/federalist-uswds-jekyll"
+  },
+  "bugs": {
+    "url": "https://github.com/18F/federalist-uswds-jekyll/issues"
+  },
   "license": "CC0-1.0",
   "dependencies": {
-    "a11y": "^0.5.1",
     "uswds": "^2.0.2"
+  },
+  "devDependencies": {
+    "a11y": "^0.5.1"
```

Comment:
This is only used locally and was slowing down the builds by downloading extra stuff, such as phantomjs!